src/overdue-libraries-upgrade.hook

```diff
@@ -1,11 +1,11 @@
 [Trigger]
 Operation = Upgrade
 Type = File
-Target = usr/lib/*
-Target = usr/lib64/*
```

Comment:
Can we leave both `lib64` and `lib32` in the `Target` list? On my system I have a symlinked `lib64` and no 32, so just keeping them all there for completeness sounds like a safe option.